### PR TITLE
Microverse html and css practice

### PR DESCRIPTION
**Pull request by belalgamal46**
Microverse html, css practice
- Create index.html and styles.css
- Setup linters and create config files
- Create gitignore
- Create navbar
- Create section
- Add Readme.md file

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @belalgamal46   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] :red_circle:  I have noticed that you have designed the wrong part of the portfolio, The project requirement requires building the landing page of your portfolio on mobile design which has two parts ( toolbar and headline )but you have designed the project card.
Kindly make sure to design the correct required design.

    - ### Your design
   image

   - ### Required design
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] [OPTIONAL] On your readme, you have the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  Kindly, make sure that The link featured in that section should point to a file at the root of your project (the license file).   Just copy it from the mentioned template:

image

That way, you will have a professional repo, and your project will be well documented 💪 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -5,7 +5,68 @@
...
+      rel="stylesheet"
+    />
+
     <link rel="stylesheet" href="./CSS/styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav id="header-container">
+        <h1 class="header">My Logo</h1>
```
Comment (ha-manel) [inline index.html]: [Optional]

- [x] It's better if the logo is in an `a` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: CSS/styles.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.menu {
+  width: 24px;
+  height: 24px;
+}
+
+/* Section */
+.hero-section {
+  background-image: url(../images/header-shapes-mobile.svg);
```
Comment (ha-manel) [inline CSS/styles.css:43]: [Optional]
- [ ] Please consider fixing the size of the `background-image`, we should try to make the website identical to Figma design 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @belalgamal46 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 
* Great design 🚀 

## Optional suggestions:
_Please check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**